### PR TITLE
Feature/mat 3729

### PR DESCRIPTION
**Pull request by RohitKandimalla**
## MADiE PR

Jira Ticket: [MAT-3729](https://jira.cms.gov/browse/MAT-3729)
(Optional) Related Tickets:

### Summary

1. Added terms and conditions Dialog on the login page 

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/oktaSignInWidget/TermsAndConditionsDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      <DialogTitle>Terms and Conditions</DialogTitle>
+      <DialogContent>
+        <DialogContentText id="alert-dialog-slide-description">
+          OMB No.0938-1236 |{" "}
+          <a href="https://www.cms.gov/regulations-and-guidance/legislation/paperworkreductionactof1995">
+            {" "}
+            Paperwork Reduction Act
+          </a>
+          <br />
+          <br />
```
Comment (adongare) [inline src/oktaSignInWidget/TermsAndConditionsDialog.tsx]: if these `<br/>` are used to separate text into paragraphs and not line breaks, I would simply use `p` tag

Comment (RohitKandimalla) [reply src/oktaSignInWidget/TermsAndConditionsDialog.tsx]: Updated it

**Review by sb-prateekkeerthi — APPROVED** [review]

**Review by adongare — APPROVED** [review]
